mgmt-lambda/handlers/updateHandler.ts

```diff
@@ -35,7 +38,25 @@ export async function handleUpdate(
     throw new ApiException(ErrorCode.LambdaFunctionNotFound)
   }
 
+  const listVersionsBeforeUpdate = await listLambdaFunctionVersions(lambdaClient, settings.LambdaFunctionName)
   const functionVersionArn = await updateLambdaFunctionCode(lambdaClient, settings.LambdaFunctionName)
+  const listVersionsAfterUpdate = await listLambdaFunctionVersions(lambdaClient, settings.LambdaFunctionName)
+
+  const newVersions = Array.from(listVersionsAfterUpdate.values()).filter(
+    (conf) => !listVersionsBeforeUpdate.has(conf.Version)
+  )
+  if (newVersions.length !== 1) {
+    console.error(`Excepted one new version, but found: ${newVersions.length} versions`)
+    throw new ApiException(ErrorCode.LambdaFunctionWrongNewVersionsCount)
+  }
+
+  const newVersion = newVersions[0]
+
+  if (newVersion.State !== State.Active) {
```

Comment:
The same question as [above](https://github.com/fingerprintjs/fingerprint-pro-cloudfront-integration/pull/211/files#r1531194033)